test/Microsoft.ML.Benchmarks/BenchmarkBase.cs

```diff
@@ -26,11 +28,38 @@ static BenchmarkBase()
         // is running is it sometime cause process hanging when the constructor trying 
         // to load MKL, this is related to below issue:
         // https://github.com/dotnet/machinelearning/issues/1073
-        public static string GetBenchmarkDataPath(string name)
+        public static string GetBenchmarkDataPathAndEncureData(string name, string path = "")
         {
```

Comment:
typo: Encure => Ensure

Reply:
Sorry, it is a different typo now. "ensure" is a single word. So it should be cased as "Ensure" and not "EnSure"

---
In reply to: [423400425](https://github.com/dotnet/machinelearning/pull/5117#discussion_r423400425) [](ancestors = 423400425)

Reply:
Yes, thanks, fixed

---
In reply to: [423402331](https://github.com/dotnet/machinelearning/pull/5117#discussion_r423402331) [](ancestors = 423402331,423400425)